ProphetBot/cogs/admin.py

```diff
@@ -148,9 +150,30 @@ async def list(self, ctx: ApplicationContext):
         name="test"
     )
     async def test(self, ctx: ApplicationContext):
```

Comment:
Not opposed to having test functionality left in, but this method should have a more descriptive name (and/or a quick comment saying what it currently does)

Reply:
Removed test command